test/controllers/update/overseas.entity.change.controller.spec.ts

```diff
@@ -57,6 +57,23 @@ describe("Overseas entity do you want to change your OE controller", () => {
       expect(resp.text).toContain(COMPANY_NUMBER);
       expect(resp.text).toContain(UPDATE_DO_YOU_WANT_TO_CHANGE_OE_TITLE);
       expect(resp.text).toContain(UPDATE_DO_YOU_WANT_TO_CHANGE_OE_NO_TEXT);
+      expect(resp.text).not.toContain(RADIO_BUTTON_NO_SELECTED);
+    });
+
+    test(`renders the ${UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE} page with radios selected to No`, async () => {
```

Comment:
Confusingly the UI has "Yes, I need to make changes" so selected Yes is noChange 0. Just rename test?

Reply:
Tests renamed